django-stubs/contrib/auth/backends.pyi

```diff
@@ -10,9 +10,7 @@ _AnyUser = Union[Model, AnonymousUser]
 UserModel: Any
 
 class BaseBackend:
-    def authenticate(
-        self, request: HttpRequest, username: Optional[str] = ..., password: Optional[str] = ..., **kwargs: Any
-    ) -> Optional[AbstractBaseUser]: ...
+    def authenticate(self, request: Optional[HttpRequest], **kwargs: Any) -> Optional[AbstractBaseUser]: ...
```

Comment:
Yeap, current `3.2.x` django agrees with this signature: https://github.com/django/django/blob/3.2.6/django/contrib/auth/backends.py#L9